src/seedu/addressbook/data/person/Address.java

```diff
@@ -37,23 +34,9 @@ public static boolean isValidAddress(String test) {
     }
 
     @Override
-    public String toString() {
-        return value;
-    }
-
-    @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof Address // instanceof handles nulls
-                && this.value.equals(((Address) other).value)); // state check
```

Comment:
Since you declared `value` as protected in the parent class so you can leave it like this, I suppose.